### DISCUSSION TITLE
DuckDNS implementation

### DISCUSSION THREAD

**Discussion by steled** [hassio-addons/addon-nginx-proxy-manager#179] [Ideas]
Hi,

I really like your addon but I'm missing an implementation for DuckDNS?
Is it planned that you will implement this as DNS Provider?

Thanks and best regards

**Comment by frenck** [comment]
I dunno, this is the add-on. You should probably ask this question @ the NGinx Proxy Manager project upstream.

(Add-ons wrap existing applications).

**Reply by steled** [reply]
Ah, sorry didn't know this.
At the project there is still a feature request, I just put a +1 there.

So the discussion can be closed.
